go.mod

```diff
@@ -1,42 +1,31 @@
 module github.com/scribe-org/scribe-server
 
-go 1.21
+go 1.23
 
 toolchain go1.23.6
 
 require (
-	github.com/getkin/kin-openapi v0.123.0
-	github.com/go-chi/chi/v5 v5.0.12
-	github.com/oapi-codegen/runtime v1.1.1
+	github.com/go-sql-driver/mysql v1.9.0
+	github.com/mattn/go-sqlite3 v1.14.24
 	github.com/spf13/viper v1.16.0
 )
```

Comment:
Ooh - looking at this made me realize.. this `go.mod` is currently serving for both `api` and `migrate` right? I'm guessing that is why the `api` dependencies got removed here? Cause populating the `go.mod` for the `migrate` dependencies cleared them?

What do we think of separating this out instead then? To then have separate `go.mod` and `go.sum` files for `api` and `migrate`